mlrun/runtimes/pod.py

```diff
@@ -633,6 +900,25 @@ def with_priority_class(self, name: typing.Optional[str] = None):
             raise mlrun.errors.MLRunInvalidArgumentError(message)
         self.spec.priority_class_name = name
 
+    def with_preemption_mode(self, mode: str):
+        """
+        Preemption modes enable users to run function pods on preemptible nodes, when filled,
+        tolerations, node labels and affinity would be populated correspondingly to the function spec on server side.
+
+        currently supports 3 modes:
+
+        allow - Allows function to be scheduled on preemptible nodes
+        constrains - Makes function to run only on preemptible nodes
```

Comment:
```suggestion
        constrains - Constrain the function to run only on preemptible nodes
```
Find how to put that only in bold (read the docs formatting)